### DISCUSSION TITLE
osu!mania Buttons on Android [Picture Sample]

### DISCUSSION THREAD

**Discussion by CamelliaNotReal** [ppy/osu#16870] [Ideas] [answered]
![Untitled11_20220214193746](https://user-images.githubusercontent.com/99637980/153857687-1da45a15-beb8-4662-9b70-5f3c4cd96b47.png)

**Comment by uzervlad** [comment, marked as answer]
#4201

**Comment by integerrr** [comment]
Stop, why are you posting the same thing again after you've been told not to do so?
